jupyter/jupyter-web-app/base_v3/kustomization.yaml

```diff
@@ -12,20 +12,12 @@
 # break the existing KFDef files. So we want to make the stacks work 
 # and then replace it.
 apiVersion: kustomize.config.k8s.io/v1beta1
-commonLabels:
```

Comment:
Why do we remove everything here?

Reply:
This is a duplication in the file, look a few lines down.